frontend/src/apis/skills.ts

```
@@ -1070,3 +1070,310 @@ export async function batchUpdateSkillsFromGit(
+  sort_order?: 'asc' | 'desc'
+}): Promise<MarketplaceSkillListResponse> {
+  const token = getToken()
+  if (!token) throw new Error('No authentication token')
+
+  const queryParams = new URLSearchParams()
+  if (params?.skip !== undefined) queryParams.append('skip', params.skip.toString())
+  if (params?.limit !== undefined) queryParams.append('limit', params.limit.toString())
+  if (params?.search) queryParams.append('search', params.search)
+  if (params?.category) queryParams.append('category', params.category)
+  if (params?.tags) {
+    params.tags.forEach((tag) => queryParams.append('tags', tag))
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)